accountsmgmt/v1/access_token_auth_builder.go

```diff
@@ -20,6 +20,8 @@ limitations under the License.
 package v1 // github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1
 
 // AccessTokenAuthBuilder contains the data and logic needed to build 'access_token_auth' objects.
+//
+//
```

Comment:
I'm not sure what is generating these changes, but it looks like too many files are touched. We'll need to figure out how to fix this before merging.

@gdbranco please take a look

Reply:
Need >= go@1.19 higher and >= goimports@0.4.0 if I'm not mistaken

`make fmt` might also resolve it, but with those versions it should work with make generate
@vkareh do you remember the command for fmt?

Reply:
I think just `make fmt` should clean them up. I get those changes every time until I remember to format it